src/screens/Home/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Text, View, Button } from 'react-native'
+import NavigationService from '../../navigationService'
+import * as firebase from 'firebase'
+
+const LogOut = async () => {
+  await firebase.auth().signOut()
```

Comment:
Watch out for asynchrony calls outside of try/catch blocks. They all should be wrapped in a try/catch block and, if you don't want to react to that error different than the original error, you can just re-throw it.
```javascript
try {
  await firebase.auth().signOut()
} catch (err) {
  throw err
}
```